prow/tide/tide.go

```diff
@@ -307,7 +306,7 @@ func (c *Controller) Sync() error {
 	var err error
 	if len(prs) > 0 {
 		start := time.Now()
-		pjList, err := c.prowJobClient.List(metav1.ListOptions{LabelSelector: labels.Everything().String()})
+		pjList, err := c.prowJobClient.List(metav1.ListOptions{})
```

Comment:
Adding the extra information that no extra information is added is IMHO primarily confusing.

If I see `LabelSelector: ` i expect a labelSelector to come and the code as it was forced me to think about this instead of being able to just go over it.

If this is important to you thought I will change it back